semanticdb-java/src/main/protobuf/semanticdb.proto

```diff
@@ -194,6 +194,7 @@ message Tree {
     // -- OUT OF SPEC -- //
     AnnotationTree annotation_tree = 9;
     AssignTree assign_tree = 10;
+    BinaryOpTree binop_tree = 11;
```

Comment:
I have always regretted using a mix of full and shortened names together in public APIs (`IdTree` probably should have been `IdentifierTree`). Either `BinopTree` or `BinaryOperatorTree` should work IMO. 
```suggestion
    BinaryOperatorTree binop_tree = 11;
```